src/woo_publications/publications/constants.py

```diff
@@ -12,3 +14,18 @@ class DocumentActionTypeOptions(models.TextChoices):
     signed = "ondertekening", _("Signed")
     received = "ontvangst", _("Received")
     declared = "vaststelling", _("Declared")
+
+
+# Taken from PLOOI documenthandelingen on
+# https://standaarden.overheid.nl/tooi/waardelijsten/expression?lijst_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fccw_plooi_documenthandelingen%2F2
+# and https://repository.officiele-overheidspublicaties.nl/waardelijsten/ccw_plooi_documenthandelingen/2/json/ccw_plooi_documenthandelingen_2.json
+DOCUMENT_ACTION_TOOI_IDENTIFIERS: Mapping[DocumentActionTypeOptions, str] = {
+    DocumentActionTypeOptions.signed: "https://identifier.overheid.nl/tooi/def/thes/kern/c_e1ec050e",
+    DocumentActionTypeOptions.received: "https://identifier.overheid.nl/tooi/def/thes/kern/c_dfcee535",
+    DocumentActionTypeOptions.declared: "https://identifier.overheid.nl/tooi/def/thes/kern/c_641ecd76",
+}
```

Comment:
These values can technically change. Do we want to do something with that? Now or in the future?

Reply:
when they change, we will update accordingly. these value lists are versioned and it's the same process if a new version of the organisations/information categories value lists is published that we need to load :)